doc/source/how-to/code/pyansys_logging.py

```diff
@@ -97,11 +94,11 @@ def log_to_stdout(self, level=LOG_LEVEL):
         self.logger = add_stdout_handler(self.logger, level=level)
         self.std_out_handler = self.logger.std_out_handler
 
-    def setLevel(self, level="DEBUG"):
+    def setlevel(self, level="DEBUG"):
```

Comment:
```suggestion
    def setlevel(self, level="DEBUG"):
```
```suggestion
    def setLevel(self, level="DEBUG"):
```
Unless I'm wrong, this is needs to be left as `setLevel` because it overwrite the default behavior of `logging`. See https://docs.python.org/3/library/logging.html#logging.Logger.setLevel